src/chocolatey.resources/helpers/functions/Get-ChocolateyUnzip.ps1

```diff
@@ -88,7 +88,8 @@ Install-ChocolateyZipPackage
 #>
 param(
   [alias("file")][parameter(Mandatory=$false, Position=0)][string] $fileFullPath,
-  [parameter(Mandatory=$true, Position=1)][string] $destination,
+  [parameter(Mandatory=$true, Position=1)]
+  [alias("unzipLocation")][string] $destination,

```

Comment:
@TheCakeIsNaOH is there a reason that you chose to break this parameter onto two lines? The other parameters in this function are all on a single line, with:

`alias - parameter - type - name`

I think it would make sense for this to remain the same format.  Thoughts?

Reply:
In `Install-ChocolateyZipPackage`, it was broken up onto two lines, so I assumed that was the format to use.

I can switch it to being on one line.

Reply:
Done.

Reply:
Gotcha.  Depending on the history of any given function, the formatting may have changed, but within any given file I think it makes sense to keep the formatting consistent.  Thanks for getting that fixed up!